src/query/src/range_select/plan_rewrite.rs

```diff
@@ -108,34 +112,77 @@ fn parse_expr_to_string(args: &[Expr], i: usize) -> DFResult<String> {
 /// Parse a duraion expr:
 /// 1. duration string (e.g. `'1h'`)
 /// 2. Interval expr (e.g. `INTERVAL '1 year 3 hours 20 minutes'`)
+/// 3. An expr can be evaluated at the logical plan stage (e.g. `INTERVAL '2' day - INTERVAL '1' day`)
```

Comment:
Seems we could use `NOW()` as an interval. This is a little confusing. Should we restrict the expression output must be interval or duration type?

Reply:
My original intention is to introduce some computing capabilities in duration param, case like `SELECT ts, min(val) RANGE (INTERVAL '2' day - INTERVAL '1' day) FROM host .....`, but duration string here(e.g. `'1h'`)  is parsed separately. The calculation capability can only be introduced on the interval type（like `INTERVAL '2' day - INTERVAL '1' day`）, but duration lacks this calculation capability (like `'2d' - '1d'`).

Reply:
Now the duration expr only support interval calculation